libs/mysql/CMakeLists.txt

```diff
@@ -4,10 +4,12 @@
 
 if (STATIC_OPENSSL)
 	if (APPLE)
-		if (${CMAKE_OSX_ARCHITECTURES} STREQUAL "i386")
-			set(OPENSSL_CONF ./Configure darwin-i386-cc "-mmacosx-version-min=${CMAKE_OSX_DEPLOYMENT_TARGET}")
-		elseif (${CMAKE_OSX_ARCHITECTURES} STREQUAL "x86_64")
+		if (${CMAKE_SYSTEM_PROCESSOR} STREQUAL "i386")
+			set(OPENSSL_CONF ./Configure $CMAKE_C_COMPILER "-mmacosx-version-min=${CMAKE_OSX_DEPLOYMENT_TARGET}")
```

Comment:
Is there a reason to use `$CMAKE_C_COMPILER` just for this line? If so, shouldn't it be `${CMAKE_C_COMPILER}`?